queries.sql

```diff
@@ -18,3 +18,41 @@ SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+/* UPDATE AND QUERY ANIMALS TABLE */
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO SAVEPOINT;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
```

Comment:
- [ ] When you are creating a `SAVEPOINT`, you have to assign a name to the `SAVEPOINT` if not it will give you this error:

![image](https://user-images.githubusercontent.com/77106962/172693740-a60b48de-e4a9-4788-ab40-e3ff366b8959.png)

**_Hint:_** You can assign a name to a `SAVEPOINT` by doing this `SAVEPOINT <name>`. Kindly implement this.